docs/howto/csrf.txt

```diff
@@ -63,29 +63,8 @@ You can acquire the token like this:
 
 .. code-block:: javascript
 
-    function getCookie(name) {
-        let cookieValue = null;
-        if (document.cookie && document.cookie !== '') {
-            const cookies = document.cookie.split(';');
-            for (let i = 0; i < cookies.length; i++) {
-                const cookie = cookies[i].trim();
-                // Does this cookie string begin with the name we want?
-                if (cookie.substring(0, name.length + 1) === (name + '=')) {
-                    cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
-                    break;
-                }
-            }
-        }
-        return cookieValue;
-    }
-    const csrftoken = getCookie('csrftoken');
-
-The above code could be simplified by using the `JavaScript Cookie library
-<https://github.com/js-cookie/js-cookie/>`_ to replace ``getCookie``:
-
-.. code-block:: javascript
-
-    const csrftoken = Cookies.get('csrftoken');
+    let tokenReg = new RegExp('csrftoken=(\\w*)');
+    const csrftoken = tokenReg.exec(document.cookie)[1];
```

Comment:
IMO, we don't need to use a regex here, see #14794 and [comment](https://github.com/django/django/pull/14794/files#r695281214) with an alternative improvement.

Reply:
The **URLSearchParams** function is intended specifically for URLs, and I dislike the idea of re-purposing it for cookies. As a one-liner, a regex is also the least code:

`const csrftoken = RegExp('csrftoken=(\\w*)').exec(document.cookie)[1];`

Reply:
> As a one-liner, a regex is also the least code:

The least is not always the best. @ngnpope Can I ask for your opinion?

Reply:
> I think there is some benefit to reducing the clunky example we have to something shorter, but I'm a little dubious about using a regex to do it. Not sure whether either of the alternatives is better? Hope this helps, Mariusz.

This helped a lot, thanks :heart: